docs/guides/kuma.md

```diff
@@ -0,0 +1,233 @@
+# Using Kuma Service Mesh with TriggerMesh
+
+[Kuma](https://kuma.io) can be configured as a Service Mesh for TriggerMesh/Knative, providing security, observability and routing among other features.
+
+The steps in this article guide you through the installation and configuration process, referring to external links when information beyond this scope is needed.
+
+## Pre-requisites
+
+Knative Serving needs to be installed on a Kubernetes cluster. We will also install Kong as the networking layer for Knative Serving since it is an ideal companion for Kuma.
+
+To install Knative Serving with Kong follow [our guide at the documentation](kong-ingress.md).
+
+This guide was written using:
+
+* Kubernetes `v1.21`
+* Knative `v1.4.0`
+* Kong Ingress Controller `2.3.1` (includes Kong `2.8`)
+* Kuma `v1.5`
+
+## Install Kuma Service Mesh
+
+Kuma can be installed using their command line tool. Follow the instructions [at the documentation](https://kuma.io/docs/1.6.x/installation/kubernetes/#download-kumactl) to install `kumactl`.
+
+Once installed execute this command to deploy Kuma Service Mesh in the cluster.
+
+```console
+kumactl install control-plane --version=1.5.0 --env-var KUMA_RUNTIME_KUBERNETES_VIRTUAL_PROBES_ENABLED=false| kubectl apply -f -
+```
+
+!!! Info "Kuma Virtual Probes"
+    The command provided at this guide disables [Kuma Virtual Probes](https://kuma.io/docs/1.6.x/policies). If you have an existing Kuma installation make sure virtual probes are disabled.
+
+Once Kuma Service Mesh is installed you should see the pod running in the `kuma-system` namespace:
+
+```console
+kumactl -n kuma-system get pods
+NAME                                 READY   STATUS    RESTARTS   AGE
+kuma-control-plane-bd98c89dc-kwj97   1/1     Running   0          19s
+```
+
+## Configure Kong
+
+We will add the label `kuma.io/sidecar-injection` to the kong namespace, so kong will be running in the mesh:
+
+```console
+kubectl label namespace kong kuma.io/sidecar-injection=enabled
+```
+
+The kong-ingress deployment needs the following annotation to be running as a Kuma gateway. The kong installation manifest used already contains this annotation.
+
+```text
+annotations: 
+  kuma.io/gateway: enabled 
+```
+
+We are going to check if it's already set:
+
+```console
+kubectl -n kong get deploy -o jsonpath='{.items[0].spec.template.metadata.annotations}'   
+
+{"kubectl.kubernetes.io/restartedAt":"2022-05-24T17:21:27+02:00","kuma.io/gateway":"enabled","traffic.sidecar.istio.io/includeInboundPorts":""}
+```
```

Comment:
Maybe we can replace all this with a command that sets the annotation with the `--overwrite` flag (just in case it already exists), and simplify all this with a single paragraph and command.

Do you think that would be possible?